src/mpid/ch4/netmod/ofi/ofi_win.h

```diff
@@ -14,6 +14,7 @@
 #include "ofi_impl.h"
 #include <opa_primitives.h>
 
+#define MAX_SHMKEY_LEN 64
```

Comment:
I assume that by namespacing you mean prefixing the name with some unique  module identifier, right? If this is the case, would something like `#define CH4_OFI_MAX_SHMKEY_LEN 64` work?